.github/workflows/compatiblity_check.yml

```diff
@@ -19,11 +19,13 @@ jobs:
 
       - name: Check Latest Dependencies
         run: |
-          git clone https://github.com/sei-protocol/sei-cosmos.git
-          git clone https://github.com/sei-protocol/sei-tendermint.git
+          git clone -b seiv2 https://github.com/sei-protocol/sei-cosmos.git
+          git clone -b seiv2 https://github.com/sei-protocol/sei-tendermint.git
           git clone https://github.com/sei-protocol/sei-iavl.git
```

Comment:
we probably don't want this change right?

Reply:
yeah good call, I'll fix this and other CI things that mentioned seiv2